app/globals.css

```diff
@@ -1,10 +1,85 @@
-.Title{
+/* Concept 1 */
+/* .card {
+  display: flex;
+  justify-content: flex-start;
+  flex-direction: column;
+  border-radius: 20px;
+  background-color: rgba(224, 193, 155, 0.441);
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+  width: 350px;
+  padding: 20px;
+  margin: 20px;
+  height: 500px;
+  font-size: 30px;
+  font-weight: 600;
+  text-align: center;
+  font-family:'Times New Roman', Times, serif;
+}
+
+.card-body {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  flex-wrap: wrap;
+  align-content: center;
+}
+
+.card-un {
+  display: flex;
+  flex-direction: column;
+  border-radius: 20px;
+  padding: 20px;
+  background-image: url('https://www.creativefabrica.com/wp-content/uploads/2023/05/08/Old-Vintage-Paper-Textured-background-Graphics-69135662-1-580x387.jpg');
+}
+
+.card-likes {
+  display: flex;
+  justify-content: flex-end;
+} */
+/* ------------------------------------ */
```

Comment:
Remove the commented code - we don't need this to be pushed up. If you want to save this for future use you can stash your changes:

Heres an example:
https://www.inmotionhosting.com/support/website/git/stash/